app/controllers/audio_files_controller.rb

```diff
@@ -31,10 +40,27 @@ def destroy
   @audio_file = AudioFile.find(params[:id])
```

Comment:
Nitpick: this doesn't need to be an instance variable; a local one will work. We're not going to have a valid audio file in `@audio_file` after we're done anyway, so it's no use to try to store it in a variable that's accessible to the view.